standards_lab/api/admin.py

```diff
@@ -1,3 +1,3 @@
-from django.contrib import admin
+# from django.contrib import admin
```

Comment:
For all the files that we are not using and we are not going to use. Would it not be better to just delete them?